### ISSUE TITLE
A11y_.NET Core_WinForms_Winforms_Containers_SplitContainer_SetBackColor_Keyboard: After pressing ENTER on the "Windows_Containers" Keyboard focus is not visible on the "button15" which is present below "SplitContainer_SetBlackColor_Panel1_Green_Panel2_Red".

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9260]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev3;#WCAG2.4.7;#DesktopApp;#Win11;#NonBenchmark;#Keyboard

### Actual:
After pressing ENTER on the "Windows_Containers" Keyboard focus is not visible on the "button15" which is present below "SplitContainer_SetBlackColor_Panel1_Green_Panel2_Red".

### Expected:
After pressing ENTER on the "Windows_Containers" Keyboard focus should be visible on the "button15" which is present below "SplitContainer_SetBlackColor_Panel1_Green_Panel2_Red".

### User Impact:
Keyboard users will not be able to understand which control has the keyboard focus after pressing ENTER on the "Windows_Containers".

### Attachments:
![Focusvisible](https://github.com/dotnet/winforms/assets/125462007/92dc1667-3200-4c21-8e39-340f78d2a4c7)

### Steps to reproduce

1. Install and Open "Winforms" App.
2. "Main Form" screen would appear.
3. TAB to "Container_Control_2: Testing the Control under the Containers tab" and press ENTER key.
4. Now observe that after pressing ENTER key on the "Container_Control_2: Testing the Control under the Containers tab"   keyboard focus is not visible on the "button15" which is present below "SplitContainer_SetBlackColor_Panel1_Green_Panel2_Red".

**Comment by merriemcgaw** [comment]
This is a Sev3 bug so we can close it on our own. @Olina-Zhang can you have your team double check TabOrder on this screen and then close this issue?

**Comment by Olina-Zhang** [comment]
@merriemcgaw I re-adjusted the controls' TabOrder, found the first control focusing is still not visible, and finally found that is caused by container's backColor, so changed it, now it is OK.
![Issue9260](https://github.com/dotnet/winforms/assets/26474449/f7a490e1-09d0-4707-adfd-c1e9c6dee26e)

**Comment by MelonWang1** [comment]
Verified this issue in the latest .NET 8.0 SDK build: 8.0.100-preview.7.23323.19 on winforms repo, it was fixed. After pressing ENTER on the "Container_Control_2" Keyboard will focus on the "button15". Screenshot as below:
![9260](https://github.com/dotnet/winforms/assets/94418985/86b93b38-cbdc-42d8-8fdc-dca8adcc083c)

**Comment by MandiMan** [comment]
Verified this on .NET 8.0 TP build: 8.0.100-preview.6.23327.6, issue was fixed, result is same as above test result.
